packages/create-bison-app/template/_templates/graphql/new/graphql.ejs

```diff
@@ -25,21 +33,32 @@ export const <%= camelized %>Queries = extendType({
   type: 'Query',
```

Comment:
while you are here, thoughts on moving this to `queryField` instead of `extendType`? Not required, but it does make it a bit more concise. 

https://nexusjs.org/docs/api/query-field

Reply:
Going to leave this one for another ticket. 
We'll want to refactor to use `query-field` and `mutation-field` throughout in our `user` module as well to stay consistent. 

I'll need to look at that syntax a bit, it looks like each piece would become its own exported `queryType`.